source/diagrams/manage-diagrams/rename-diagrams.txt

```diff
@@ -10,4 +10,47 @@ Rename Diagrams
    :depth: 1
    :class: singlecol
 
-Rename diagrams.
\ No newline at end of file
+You can rename diagrams on the Mapping tab of Relational Migrator.
+Relational Migrator displays diagram names on the ribbon tab. 
+
+About this Task
+---------------
+
+You can rename diagrams to customize your diagram properties. Relational
+Migrator shows your diagram names on the ribbon tab of the Mapping view and in
+the Diagram Properties pane. 
+
+Steps
+-----
+
+.. procedure::
+   :style:  connected
+
+   .. step:: Navigate to the diagram you want to rename. 
+
+      Click the name of the diagram in the ribbon tab to display the
+      diagram. 
+   
+   .. step:: Open the Diagram Properties pane.
+
+      Click the arrow button on the right of your window to open the
```

Comment:
[optional suggestion] Assuming that the arrow button is the one in the screenshot, it might help to add an icon role so they know exactly what arrow to look for. For this one I'd suggest :icon-fa5:`chevron-circle-right`"

```suggestion
      Click the :icon-fa5:`chevron-circle-right` arrow button on the right of your window to open the
```

Reply:
Done!